mwclient/page.py

```diff
@@ -478,10 +483,11 @@ def links(self, namespace=None, generator=True, redirects=False):
                                                  return_values='title', **kwargs)
 
     def revisions(self, startid=None, endid=None, start=None, end=None,
-                  dir='older', user=None, excludeuser=None, limit=50,
+                  dir='older', user=None, excludeuser=None, limit=None,
                   prop='ids|timestamp|flags|comment|user',
                   expandtemplates=False, section=None,
-                  diffto=None, slots=None, uselang=None):
+                  diffto=None, slots=None, uselang=None, max_items=None,
+                  api_chunk_size=None):
```

Comment:
Should the `api_chunk_size` have a default value of 50 here, to match the old `limit` default?

```suggestion
                  api_chunk_size=50):
```